akka-persistence/src/main/scala/akka/persistence/PersistentActor.scala

```diff
@@ -21,6 +21,14 @@ case object RecoveryCompleted extends RecoveryCompleted {
   def getInstance = this
 }
 
+/** Convenience for skipping recovery of [[PersistentActor]] instances. */
+object NoRecovery {
```

Comment:
Bikeshedding on naming... `def recover = Recover.none` looks ok, but not sure if none is a good word here?